### ISSUE TITLE
ink-docstrap dependency needs to be updated

### ISSUE THREAD

**Issue by sullivanpt** [gamtiq/grunt-init-pack#1]
grunt-jsdoc no longer incudes ink-docstrap. To get the build to work I added this to my package.json:

```
    "ink-docstrap": "^0.5.2",
```

and changed the template path in the Gruntfile.js

```
                    template: "node_modules/ink-docstrap/template",
```

**Comment by gamtiq** [comment]
Thank you for the information!
I will try to fix this issue asap.

**Comment by gamtiq** [comment]
This issue is fixed in [version 0.2.2](https://github.com/gamtiq/grunt-init-pack/releases/tag/0.2.2).
